src/elements/TheAlertModal/TheAlertModal.tsx

```diff
@@ -15,48 +16,90 @@ import styles from './TheAlertModal.styles';
 
 export default () => {
 	const [ globalState, globalActions ] = useGlobal() as any;
-	const { alert }: { alert: Alert } = globalState;
+	const { alert: alertObj }: { alert: Alert } = globalState;
 	const { clearAlert } = globalActions;
-
 	const scheme = useScheme();
 
 	const handleCloseButtonPress = () => {
 		clearAlert();
 	};
 
 	const handleDismiss = () => {
-		if (alert.dismissable) {
+		if (alertObj.dismissable) {
 			clearAlert();
 		}
 	};
-	return (
-		<Modal
-			style={styles.container}
-			title={alert?.title || 'Alert'}
-			palette="accent"
-			open={alert !== undefined}
-			onDismiss={handleDismiss}
-		>
-			<View style={styles.body}>
-				<View style={styles.textContainer}>
-					<Text style={typography.body1}>
-						{alert?.message || 'Uh oh, an unknown error occurred!'}
-					</Text>
-				</View>
+	if (!alertObj) return null;
+	if (!alertObj.type || alertObj.type === 'default') {
+		return (
+			<Modal
+				style={styles.container}
+				title={alertObj?.title || 'Alert'}
+				palette="accent"
+				open={alertObj !== undefined}
+				onDismiss={handleDismiss}
+			>
+				<View style={styles.body}>
+					<View style={styles.textContainer}>
+						<Text style={typography.body1}>
+							{alertObj?.message || 'Uh oh, an unknown error occurred!'}
+						</Text>
+					</View>
 
-				<TextButton
-					text="OK"
-					style={{
-						width: 104,
-					}}
-					buttonStyle={{
-						default: scheme.primary,
-						pressed: scheme.secondary,
-						disabled: scheme.disabled,
+					<TextButton
+						text="OK"
+						style={{
+							width: 104,
+						}}
+						buttonStyle={{
+							default: scheme.primary,
+							pressed: scheme.secondary,
+							disabled: scheme.disabled,
+						}}
+						onPress={handleCloseButtonPress}
+					/>
+				</View>
+			</Modal>
+		);
+	}
+	if (alertObj.type === 'coming soon') {
+		return (
+			<Modal
+				style={styles.container}
+				title={alertObj?.title || 'COMING SOON!'}
+				palette="secondary"
+				open={alertObj !== undefined}
+				onDismiss={handleDismiss}
+			>
+				<View style={styles.body}>
+					<Icon name="smile" size={75} />
+					<View style={{
+						padding: 5,
+						flexGrow: 1,
+						alignItems: 'center',
+						justifyContent: 'center',
```

Comment:
It would be better to refactor this into a new component called `ComingSoonModal` for readability

Reply:
Got it. That's a good suggestion. I will modify my code.